src/main/java/com/faforever/client/chat/ChatUserItemController.java

```diff
@@ -306,31 +306,12 @@ private void setClanTag(String clanTag) {
     updateClanMenu();
   }
 
-  private void updateGameStatus() {
-    if (chatUser == null) {
-      return;
-    }
-    Optional<Player> playerOptional = chatUser.getPlayer();
-    if (!playerOptional.isPresent()) {
-      statusLabel.setText("");
-      return;
-    }
-
-    Player player = playerOptional.get();
-    switch (player.getStatus()) {
-      case IDLE:
-        statusLabel.setText("");
-        break;
-      case HOSTING:
-        statusLabel.setText(i18n.get("user.status.hosting", player.getGame().getTitle()));
-        break;
-      case LOBBYING:
-        statusLabel.setText(i18n.get("user.status.waiting", player.getGame().getTitle()));
-        break;
-      case PLAYING:
-        statusLabel.setText(i18n.get("user.status.playing", player.getGame().getTitle()));
-        break;
-    }
+  private static void loadPlayerStatusIcons() throws IOException {
+    playerStatusIcons = new HashMap<>();
+    playerStatusIcons.put(PlayerStatus.HOSTING, new Image(new ClassPathResource("/images/player_status/host.png").getURL().toString(), true));
+    playerStatusIcons.put(PlayerStatus.LOBBYING, new Image(new ClassPathResource("/images/player_status/lobby.png").getURL().toString(), true));
+    playerStatusIcons.put(PlayerStatus.PLAYING, new Image(new ClassPathResource("/images/player_status/playing.png").getURL().toString(), true));
+//    playerStatusIcons.put(PlayerStatus.PLAYING_NOT_SPECTATEABLE, new Image(new ClassPathResource("/images/player_status/playingNotSpectateable.png").getURL().toString(), true));
```

Comment:
commeted out code

Reply:
yes, that's intentional, this is so far not supported by the client

Reply:
Then make a comment explaining that